src/main/java/com/hei/wallet/wallety/endpoint/rest/controller/TransferController.java

```diff
@@ -0,0 +1,6 @@
+package com.hei.wallet.wallety.endpoint.rest.controller;
+import com.hei.wallet.wallety.model.Transfer;
+
+public class TransfertController {

```

Comment:
I think you can try to implement tbe crud without any features for Transfer... like getAll, getById...and save